blocks/inspector-controls/button-control/index.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * WordPress dependencies
+ */
+import { Button, withInstanceId } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import BaseControl from './../base-control';
+import './style.scss';
+
+function ButtonControl( { instanceId, label, value, help, ...props } ) {
+	const id = 'inspector-button-control-' + instanceId;
```

Comment:
Is the instanceId and the id useful in this component? Generally, we're using it to match labels and inputs but there's no input in this component.

Reply:
Good catch. `<button>` elements [are labelable](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Form_labelable), but I wasn't passing `id` along to the `<Button>` component. I've fixed this in 224eb27f54a397b3c64f3793174fcb9c2e0cd252.